components/DateInput.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import {
+  Controller,
+  Control,
+  useFormState,
+  FieldError,
+} from 'react-hook-form';
+import { LocalizationProvider, DatePicker } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+
+type DateInputProps = {
+  name: string;
+  control: Control;
+  label?: string; // eslint-disable-line
```

Comment:
Why did you disable the eslint rule for these line?

Reply:
If I try to define defaultProps I get this warning
<img width="666" alt="image" src="https://github.com/javiercodixis/whiskeyjs/assets/143386814/5a654bb7-82a1-42a6-8ca8-d6febffc4105">

Reply:
And if you do, what you did in the TextInput (rules = {} - defining the default value on the prop), you still get the error?

Reply:
if I remove this eslint comment 
I get same warning to make defaultProps, which will lead to same warning after the solution of creating defaultProps

<img width="866" alt="image" src="https://github.com/javiercodixis/whiskeyjs/assets/143386814/89af8660-6c24-42ef-85f1-dafdc9c439fc">